api/v1alpha2/types.go

```diff
@@ -469,9 +476,3 @@ type SubnetSpec struct {
 	// SecurityGroup defines the NSG (network security group) that should be attached to this subnet.
 	SecurityGroup SecurityGroup `json:"securityGroup,omitempty"`
 }
-
-const (
-	AnnotationClusterInfrastructureReady = "azure.cluster.sigs.k8s.io/infrastructure-ready"
```

Comment:
These three constants were not referenced anywhere in CAPZ code, nor were the annotation strings themselves.

Reply:
Whoops, reverting that one.